test/metabase/models/setting_test.clj

```diff
@@ -199,6 +199,30 @@
               #"Cannot initialize setting before the db is set up"
               (setting/get :test-setting-custom-init)))))))
 
+(def ^:private base-options
+  {:setter   :none
+   :type     :csv
+   :default "base-default"})
+
+(defsetting test-base-setting
+  "Setting to test the `:feature` property of settings. This only shows up in dev."
+    :visibility :internal
+    :default "my,default"
+    :base base-options)
+
+(deftest ^:parallel defsetting-with-base-test
+  (testing "A setting which specifies some base options"
```

Comment:
```suggestion
  (testing "A setting which specifies a base option"
```

Reply:
I prefer the original wording here, it is very intentional. The way I use the terms:

- Each key value pair is one option
- The base map is therefore a collection of options, or "base options"
- We are specifying "some" base options, as it is optional.

Reply:
makes sense to me